optuna/integration/tensorboard.py

```diff
@@ -56,12 +56,7 @@ def __call__(self, study: optuna.study.Study, trial: optuna.trial.FrozenTrial) -
     def _add_distributions(
         self, distributions: Dict[str, optuna.distributions.BaseDistribution]
     ) -> None:
-        real_distributions = (
-            optuna.distributions.UniformDistribution,
-            optuna.distributions.LogUniformDistribution,
-            optuna.distributions.DiscreteUniformDistribution,
-            optuna.distributions.FloatDistribution,
-        )
+        real_distributions = (optuna.distributions.FloatDistribution,)
```

Comment:
[note] We can simply rewrite `supported_distributions` as `[FloatDistribution, IntDistribution, CategoricalDistribution]`, I think. It would be follow-up because cleanup `IntDistribution` is also ongoing.